pkg/skaffold/sync/kubectl/kubectl.go

```diff
@@ -60,18 +58,30 @@ func (k *Syncer) Sync(ctx context.Context, s *sync.Item) error {
 	return nil
 }
 
-func deleteFileFn(ctx context.Context, pod v1.Pod, container v1.Container, src, dst string) []*exec.Cmd {
-	delete := exec.CommandContext(ctx, "kubectl", "exec", pod.Name, "--namespace", pod.Namespace, "-c", container.Name, "--", "rm", "-rf", dst)
+func deleteFileFn(ctx context.Context, pod v1.Pod, container v1.Container, files map[string]string) []*exec.Cmd {
+	// "kubectl" is below...
+	deleteCmd := []string{"exec", pod.Name, "--namespace", pod.Namespace, "-c", container.Name, "--", "rm", "-rf"}
+	args := make([]string, 0, len(deleteCmd)+len(files))
+	args = append(args, deleteCmd...)
+	for _, dst := range files {
+		args = append(args, dst)
+	}
+	delete := exec.CommandContext(ctx, "kubectl", args...)
 	return []*exec.Cmd{delete}
```

Comment:
Both copyFn and deleteFn can just return a single exec.Cmd? WDYT?

Reply:
No harm in leaving them as slices, but I'm indifferent.